### PR TITLE
Climber and Climb

### PR DESCRIPTION
**Pull request by Yoel-Perman-Brilliant**

### CODE REVIEW

File: src/main/java/frc/robot/commands/Climb.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import frc.robot.subsystems.Climber;
+
+public class Climb extends ParallelCommandGroup {
+
+    public Climb(Climber leftClimber, Climber rightClimber) {
```
Comment (TurtleMinecraft) [inline src/main/java/frc/robot/commands/Climb.java]: make climbers close after opening

File: src/main/java/frc/robot/subsystems/Climber.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+public class Climber extends DoubleSolenoidSubsystem {
+
+    private static final String LEFT_NAMESPACE_NAME = "left climber";
+    private static final String RIGHT_NAMESPACE_NAME = "right climber";
+
+    private static final boolean LEFT_INVERTED = false;
+    private static final boolean RIGHT_INVERTED = false;
+
+    private static Climber leftInstance, rightInstance;
```
Comment (TurtleMinecraft) [inline src/main/java/frc/robot/subsystems/Climber.java]: seperate

**Review by TurtleMinecraft — DISMISSED** [review]

File: src/main/java/frc/robot/commands/OpenClimb.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import frc.robot.subsystems.Climber;
+
+public class OpenClimb extends ParallelCommandGroup {
```
Comment (Yoel-Perman-Brilliant) [inline src/main/java/frc/robot/commands/OpenClimb.java]: OpenClimber

File: src/main/java/frc/robot/commands/CloseClimb.java [hunk 4]
```diff
@@ -0,0 +1,11 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import frc.robot.subsystems.Climber;
+
+public class CloseClimb extends ParallelCommandGroup {
```
Comment (Yoel-Perman-Brilliant) [inline src/main/java/frc/robot/commands/CloseClimb.java]: CloseClimber

**Review by Spikes-2212 — APPROVED** [review]

**Pull request merged**